src/main/java/duke/utils/Storage.java

```diff
@@ -0,0 +1,52 @@
+package duke.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class Storage {
+
+    private String fileName;
+
+    public Storage(String fileName) {
+        this.fileName = fileName;
+    }
+
+    public void createStorage() throws IOException {
+        //relative path
+        String relativePath = fileName;
+        File file = new File(relativePath);
+        if (file.createNewFile()) {
+            System.out.println(relativePath + " Save File Created");
```

Comment:
For printing, we can always define a print(String s) method which makes the code much shorter and less frustrating to type.